Gemfile

```
@@ -22,7 +22,7 @@ gem 'flipper-active_record', '~> 0.20.3'
 gem 'flipper-active_support_cache_store', '~> 0.20.3'
 gem 'flipper-ui', '~> 0.20.3'
 # Use Hotwire to make the app feel like an SPA
-gem 'hotwire-rails', '~> 0.1.0'
+gem 'hotwire-rails', '~> 0.1.3'
```

```suggestion
gem 'hotwire-rails', '~> 0.1'
```

Should be better right?

---

yes seems good to me!